README.md

```diff
@@ -45,56 +23,145 @@ aws_secret_access_key=xxxxxxxx
         "gs_bucket_prefix": "gdc-varepop-apollo-phs001374"},
 }
 ```
-- Upload the `manifest` to S3 bucket `s3://INPUT_BUCKET/input/`.
-- Run `jobs/kube-script-setup.sh`.
-An example command to run the aws sync job with 100 threads and each thread handles 50 files.
+
+## Manifest Files
+### Manifest Formats
+Both the `sync manifest` and `redaction manifest` have the same format as described below:
 ```
-gen3 runjob jobs/aws-bucket-replicate-job.yaml GDC_BUCKET_NAME gdcbackup MANIFEST_S3 s3://tests/manifest THREAD_NUM 100 LOG_BUCKET log_bucket CHUNK_SIZE 50
+id	file_name	md5	size	state	project_id	baseid	version	release	acl	type	deletereason	url
+ada53c3d-16ff-4e2a-8646-7cf78baf7aff	ada53c3d-16ff-4e2a-8646-7cf78baf7aff.vcf.gz	ff53a02d67fd28fcf5b8cd609cf51b06	137476	released	TCGA-LGG	6230cd6d-8610-4db4-94f4-2f87592c949b	1	12.0	[u'phs000xxx']	active		s3://xxxx/ada53c3d-16ff-4e2a-8646-7cf78baf7aff
+2127dca2-e6b7-4e8a-859d-0d7093bd91e6	2127dca2-e6b7-4e8a-859d-0d7093bd91e6.vep.vcf.gz	c4deccb43f5682cffe8f56c97d602d08	136553	released	TCGA-LGG	6a84714f-2523-4424-b9cb-c21e8177cd0f	1	12.0	[u'phs000xxx']	active		s3://xxxx/2127dca2-e6b7-4e8a-859d-0d7093bd91e6/2127dca2-e6b7-4e8a-859d-0d7093bd91e6.vep.vcf.gz
 ```
 
-### GOOGLE sync
-- Put the `gcloud-creds-secret` and `dcf_dataservice_settings` into the `dcf_dataservice` folder. 
```

Comment:
Why did you remove this. Its not part of the current process but this instruction is required for initial set up

Reply:
if you read you'll find it